### DISCUSSION TITLE
i3 freezes computer when launching a second TTY

### DISCUSSION THREAD

**Discussion by dojero** [i3/i3#4628] [Q&A] [answered]
Booting Arch into tty1 and then running startx to launch X and openbox for user1. From user1, invoking tty2 with ctrl-alt-F2. Log in user2 and then run startx to launch X and i3 (user2 uses i3, user1 uses openbox). User2 is delivered to workspace 1, but the system is completely frozen. No keys work. The only solution is to long press the power button and shut down. From an initial boot, user2 can log in and launch i3 without incident on tty1. BTW, if I set user2 up to launch openbox instead of i3, it works perfectly in the tty2 scenario. That's why I'm guessing this is an i3 issue. 

Any ideas for me as to why this is happening and how I might fix it?

**Comment by orestisfl** [comment, marked as answer]
i3 does not do this (unless if we introduced a bug since I last did this)

Can you start from an empty xinitrc and give it a try? Meaning only `exec i3`. If you have `dbus-launch` somewhere in there it is known to introduce problems without any benefit. 

 Also remove all other startup commands from your i3 config or other places.

**Reply by dojero** [reply]
Converting to an .xinitrc with only a single line for executing i3 worked. Incidentally, the .xinitrc for that user did not contain the dbus-launch (or any other dbus reference). Which means that I'll have to try to figure out what is causing the problem, but, in the meantime, this is working. Thanks.
